tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Framework.cs

```diff
@@ -511,6 +511,17 @@ public Dictionary<string, object> GetResponseHeadersForWaf()
         return headersDic;
     }
 
+    internal static void CollectHeaders(Span internalSpan)
+    {
+        var context = HttpContext.Current;
```

Comment:
I guess this is safe to call, even if you're in a console app or something? Have you tested it, or are we already doing this elsewhere anyway?

Reply:
yes it doesn't crash from the Samples.ManualInstrumentation in Net4.6.2, it just returns null